### PR TITLE
Updates to Managed OS Patching / FAQ Docs

### PR DESCRIPTION
**Pull request by bjswoboda**
Per request from Chris Little:
https://t3n.zendesk.com/agent/tickets/1186484.

### CODE REVIEW

**Comment by nathanyoung** [comment]
Need to update your branch with master. How to: https://github.com/CenturyLinkCloud/PublicKB#merging-an-upstream-repository-into-your-fork

File: Managed Services/managed-os-operating-system-patching-and-update-processes.md [hunk 1]
```diff
@@ -3,24 +3,59 @@
...
+
+### Service Packs and Sub Releases
+
+All service packs are evaluated and tested thoroughly on the standard CenturyLink Cloud Server configurations. Integration of the service pack into the CenturyLink standard operating system build only occurs once testing has proven its stability and performance benefits. All applicable servers will be automatically configured with this new base build after it has completed certification.
+
+CenturyLink intends to keep in production current level N-2 to the extent that the underlying vendor is still providing patching and support. When underlying vendor support ceases to exist, CenturyLink  will be liable for best effort service and patching up to the last available support structure. Existing production servers, if requested by the customer, will be upgraded during maintenance windows that will be coordinated with the customer.
+
+### Anti-Virus Protection
```
Comment (chrislittle) [inline Managed Services/managed-os-operating-system-patching-and-update-processes.md]: is there a reason this whole statement on AV is in a managed OS patching KB.  i don't see how they are related in any way.  they are completely independent in my view.  but let me know.

**Comment by bjswoboda** [comment]
It had always been there. I assume clients had been interested in how AV was updated. I was not looking at AV as part of the scope of this work.

**Comment by chrislittle** [comment]
thx no worries

**Pull request merged**